packages/twenty-front/src/modules/object-record/record-filter/types/RecordFilterDefinitionByEntity.ts

```diff
@@ -0,0 +1,5 @@
+import { RecordFilterDefinition } from './RecordFilterDefinition';
+
+export type RecordFilterDefinitionByEntity<T> = RecordFilterDefinition & {
```

Comment:
let's remove Entity naming too not sure what this ByEntity means

Reply:
Turned out to be an orphan file.